### ISSUE TITLE
After running "create new styles", the plugin appends a " 1" to every style name in Sketch 66.1

### ISSUE THREAD

**Issue by renderghost** [franklymrshankly/Create-New-Styles#2]
**These are the layer names**
e.g. `type/@grey/48-semibold`

![image](https://user-images.githubusercontent.com/21337875/84894262-cd123580-b0a0-11ea-92ef-9e7262389620.png)

**Here are the results**
e.g. `type/@grey/48-semibold 1`

![image](https://user-images.githubusercontent.com/21337875/84894073-8b818a80-b0a0-11ea-964d-2bbb482a3743.png)

This occurs in new documents with no other styles **in the document or linked libraries**.

Very strange and very frustrating. Any ideas?

Thanks so much.

**Comment by franklymrshankly** [comment]
Hi Barry,

Thank you for submission.

This behaviour is intentional, as the plugin requires a number at the end of the string to create a new version number. For example if there was only one style in the document named "mystyle 15" after running the plugin, a new generated style would be named "mystyle 16". I've tried looking into the problem but I haven't managed to solve it.

I've now moved away from JS, so I'm happy for anyone else to push/fork a solution.

Many apologies.
